Quick-Start [DA]/pg_cext.md

```diff
@@ -0,0 +1,148 @@
+# External Languages in PostgreSQL:
+External languages in PostgreSQL provide a powerful way to integrate additional programming languages, extending the database's capabilities beyond its native support. Developers can use these languages to implement custom functionality, access external libraries, and leverage their expertise in preferred programming languages.
+
+Using external languages offers several advantages:
+
+1. **Language Flexibility:** Integrate a wide range of languages like Python, Ruby, or Java to perform tasks that may not be as straightforward in SQL.
+
+2. **Code Reusability:** Reuse existing code written in preferred languages, reducing development time and effort.
+
+3. **Performance Optimization:** Choose languages with performance advantages for specific tasks, improving the overall extension efficiency.
+
+4. **Ecosystem Integration:** Tap into well-established language ecosystems, leveraging community-contributed libraries and best practices.
+
+To utilize external languages in a PostgreSQL extension, follow these steps:
+
+1. **Implement the Language:** Develop the code and functionality to support the external language within the extension.
```

Comment:
I don't follow this. I don't have to implement C to build a C extension. A little confused.

Reply:
I did create a simple C file to implement the sum function, this point is to just showcase that point.